### DISCUSSION TITLE
Support `Pull Requests` and `Features` property changes in the `repository` hook

### DISCUSSION THREAD

**Discussion by matuella** [community/community#40280] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

Currently, there is no way to get notified (through `repository` Webhooks) when any setting of Pull Requests or Features tab changes (can be found in the General tab of any repository settings). I would expect that when someone changes a property like "Allow Merge Commits", it would send me an `edited` action type with the changed property value.

**Comment by ossanna16** [comment]
Hi there @matuella and welcome to our community! Thank you for the feedback!
